docker-compose.yml

```diff
@@ -129,8 +128,10 @@ services:
       ASPNETCORE_URLS: https://+:443
       ASPNETCORE_Kestrel__Certificates__Default__Password: ndsa
       ASPNETCORE_Kestrel__Certificates__Default__Path: /https/aspnetapp.pfx
-      API_BASEADDRESS: http://api:5002
-      # API_BASEADDRESS: http://mock-api:5555
+      SiteApi__BaseUrl: http://api:5002
+      SupplierApi__BaseUrl: http://api:5002
+      AgreementApi__BaseUrl: http://api:5002
+      OrganisationApi__BaseUrl: https://directory.spineservices.nhs.uk
```

Comment:
The `mock-ods` service in this stack also fulfils this role

Could we either ..

- use the `uat` ODS server *or*
- use the `mock-ods` service (preferred)
  - This is currently not a dependency of this service, so that needs adding
  - The mock ODS service only has one set of data items in it, will add docs on these and how to add more to this branch